common-practices-tools/security/yubikey.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Using your YubiKey
+---
+
+Notes on installing and setting up your _YubiKey_ for various platforms and applications.
+
+## Introduction
+
+The YubiKey is a hardware device manufactured by Yubico that provides a hardware, phishing resistant "second factor" enabling true multi-factor authentication: something you know (your password) and something you have (your YubiKey). It enables you to easily and securely log in to accounts by emitting one-time passwords or using a FIDO-based public/private key pair generated by the device.
+
+### Operation
+
+-   Simply plug it into an unused USB port.
+-   During certain types of authentication you will be prompted on screen to press the inset copper button marked with and (often lit) "Y".
+
+### Security Hints
+
+-   If you trust your environment (like at home) you can keep the YubiKey near or even plugged into your computer.
+-   In low trust environments (coffee shops, hotel rooms, etc.) keep your YubiKey with you at all times (in a pocket or purse). If your computer is compromised, it won't be accessible without the YubiKey that you have on you.
+-   Do not use SMS text messages for multi-factor authentication. Your MFA is only as strong as the weakest method you have configured.
+
+## Enable YubiKey MFA for applications
+
+See "[Let's get your YubiKey to work](https://yubico.com/start)" (from Yubico) on how to use MFA with: [Gmail](https://www.yubico.com/why-yubico/for-individuals/gmail-for-individuals), [LastPass](https://www.yubico.com/works-with-yubikey/catalog/lastpass-premium-and-families/), [GitHub](https://www.yubico.com/works-with-yubikey/catalog/github/) and many other services.
+
+_See also: [How to Secure Your Google, Dropbox, and GitHub Accounts With a Yubikey](http://www.howtogeek.com/232360/how-to-secure-your-google-dropbox-and-github-accounts-with-a-u2f-key/)_
+
+### LastPass
+
+This requires a YubiKey token (cover the button for approximately one second) on laptop/desktop to unlock LastPass.
+
+<!-- prettier-ignore-start -->
+
+- My Vault -> Account Settings -> Multifactor Options
+  - Set up one free option (e.g., [Google Authenticator](https://support.google.com/accounts/answer/1066447?hl=en)) - this is a useful backup
```

Comment:
On https://guidebook.civicactions.com/en/latest/common-practices-tools/security/#multi-factor-authenticators-mfa we say:
> Note that Google Authenticator is no longer recommended as it does not support encrypted cloud backup.

...which I believe is still the case. (Should also say we prefer the work-alike Authy.)
Can we switch this to "Authy" and also make the change on line 60 below?

Reply:
Just found: https://security.googleblog.com/2023/04/google-authenticator-now-supports.html -- so maybe should remove that line from the Awareness and Tools page... though I'm a bit uneasy (for some reason) about the passwords becoming part of a Google account. Thoughts?

Reply:
How about this @openprivacy - I removed the note about Google Authenticator not being recommended, since it now supports cloud backup. But we won't list it as one of the recommended 3 that are currently there (LP Authenticator, Authy & Yubikey). I also added Lastpass Authenticator to row 35 of the yubikey.md page above, in addition to Google Authenticator. I think as long as users don't store their actual passwords in the Google Password Manager/Chrome, then backing up the authenticator app to Google doesn't add much risk. An adversary would still need their password. I haven't heard of any system compromises due to getting access to an authenticator app's backup info.

Reply:
Per the conversation in the internal security TEM, add a note about not leaving your Yubikey in your laptop if you are working away from home and step away from it (Starbucks bathroom scenario).